src/view/renderer.js

```diff
@@ -182,7 +189,8 @@ export default class Renderer {
 		// There was inline filler rendered in the DOM but it's not
 		// at the selection position any more, so we can remove it
 		// (cause even if it's needed, it must be placed in another location).
-		if ( this._inlineFiller && !this._isSelectionInInlineFiller() ) {
+		// Filler should not be touched during composition to not break it.
+		if ( this._inlineFiller && !this.isComposing && !this._isSelectionInInlineFillerOnlyNode() ) {
```

Comment:
Could you explain the change from `isSelectionInInlineFiller()` to `isSelectionInInlineFillerOnlyNode()`?

Reply:
Described it in more detail in comment above - https://github.com/ckeditor/ckeditor5-engine/pull/1355#issuecomment-375609711 in **Adjusting the general behaviour when filler is removed** section.